Sources/Graphics/context/GraphicsContext.swift

```diff
@@ -1,13 +1,21 @@
 import Utils
 
+public enum PixelFormat {
+    case rgba32
+    case g8
+}
+
 /** A stateful 2D drawing environment. */
 public protocol GraphicsContext {
     associatedtype Image: Sized
     associatedtype SVG: Sized
 
-    /** Creates a new context with the given width and height. */
+    /** Creates a new context with the given width and height in RGBA32. */
     init(width: Int, height: Int) throws
```

Comment:
That's better! Sorry I failed to spot that. I made a new extension alongside the current one so that they're someone focussed - I can imagine over time more constructors being added as more underlying options are exposed. But that does look a bit silly just having the one, so if you don't like it I can squash it down to one extension - I just like to have my extensions focussed.